scroll.js

```diff
@@ -0,0 +1,14 @@
+window.onscroll = function () {
```

Comment:
You don't need to use a callback here, just put the name of the function without parenthesis.